src/aims_ui/page_error.py

```diff
@@ -5,13 +5,18 @@
 def page_error(
     api_response,
     page_name,
+    connection_error=False,
 ):
 
-  if len(api_response.json().get('errors', [])) > 0:
-    error_description = api_response.json().get('errors',
-                                                [''])[0].get('message')
+  if not connection_error:
```

Comment:
There's some repetition here. You can store `api_response.json().get('errors', [])` into a variable and re-use it, instead.

Reply:
Fixed